tools/cldr-apps/src/main/webapp/js/new/cldrAdmin.js

```diff
@@ -177,7 +177,7 @@ const cldrAdmin = (function () {
           cldrDom.createChunk("Session: " + sess, "span", "adminUserSession")
         );
         if (cs.user) {
-          user.appendChild(cldrListUsers.createUser(cs.user));
+          user.appendChild(cldrAdmin.createUser(cs.user));
         } else {
```

Comment:
almost everything moved from cldrListUsers to cldrAdmin in the previous pr, but I didn't remember to change this one then

webpack actually pointed out the error